data/json/items/gun/converted.json

```diff
@@ -90,6 +90,27 @@
     "name": { "str": "AR-15" },
     "default_mods": [ "retool_ar15_762" ]
   },
+  {
+    "id": "modular_deagle_50",
```

Comment:
i recommend to do the same #69725 did, and change three separate guns with default mods to three itemgroups with mod installed

```json
  {
    "id": "modular_deagle_50",
    "type": "item_group",
    "subtype": "collection",
    "entries": [ { "item": "modular_deagle", "contents-item": "retool_deagle_50" } ]
  },
```